src/main/java/com/terraformersmc/terrestria/init/TerrestriaFeatures.java

```diff
@@ -172,7 +187,7 @@ public static void init() {
 				new VolcanoStructureFeature(DefaultFeatureConfig::deserialize)
 		);
 
-		Feature.STRUCTURES.put("Volcano", VOLCANO_STRUCTURE);
+		Feature.STRUCTURES.put("volcano", VOLCANO_STRUCTURE);
```

Comment:
This may break worlds! We need to verify that it doesn't

Reply:
I'll label this as todo for the future